charts/ts-translation-service/values.preview.template.yaml

```diff
@@ -2,3 +2,17 @@ java:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  environment:
+    TS_TRANSLATION_SERVICE_DB_HOST: "{{ .Release.Name }}-postgresql"
+    TS_TRANSLATION_SERVICE_DB_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
+    TS_TRANSLATION_SERVICE_DB_USERNAME: "{{ .Values.postgresql.postgresqlUsername}}"
+    TS_TRANSLATION_SERVICE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
+    TS_TRANSLATION_SERVICE_DB_OPTIONS: "?stringtype=unspecified"
+    TS_TRANSLATION_SERVICE_DB_PORT: 5432
+  postgresql:
+    enabled: true
+  keyVaults:
+    ts:
+      resourceGroup: ts-shared
+      secrets:
+        - name: AppInsightsInstrumentationKey
```

Comment:
I suspect we need to also add `IDAM_KEY_TS_TRANSLATION_SERVICE` and `TS_TRANSLATION_SERVICE_IDAM_CLIENT_SECRET` for preview to work fully.

Reply:
Neither of these values exist yet in the `ts-aat` vault, so will add in a separate PR